modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -5,8 +5,23 @@
 [id="nw-ovn-kubernetes-limitations_{context}"]
 = OVN-Kubernetes limitations
 
-The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has a limitation that is related to traffic policies.
-The network provider does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
+The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
+
+* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
 The default value, `cluster`, is supported for both parameters.
 This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
 
+// The foll limitation is also recorded in the installation section.
+* For clusters configured for dual-stack networking, both IPv4 and IPv6 traffic must use the same network interface as the default gateway.
+If this requirement is not met, pods on the host in the `ovnkube-node` daemon set enter the `CrashLoopBackOff` state.
+If you display a pod with a command like `oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node -o yaml`, the `status` field contains more than one message about the default gateway:
```

Comment:
I suggest adding "as shown in the following output"

Reply:
Absolutely.  Sorry that I missed it.